awx/main/models/events.py

```diff
@@ -794,13 +798,18 @@ class Meta:
         editable=False,
         db_index=False,
     )
+    # We need to keep this as a FK in the model because AdHocCommand uses a ManyToMany field
+    #   to hosts through adhoc_events. But in https://github.com/ansible/awx/pull/8236/ we
+    #   removed the nulling of the field in case of a host going away before an event is saved
+    #   so this needs to stay SET_NULL on the ORM level
     host = models.ForeignKey(
         'Host',
         related_name='ad_hoc_command_events',
         null=True,
         default=None,
         on_delete=models.SET_NULL,
```

Comment:
Why is this one still `SET_NULL`?

Reply:
I think initially, when I was going for IntegerFields, I was just going to leave this field alone because it had to remain a ForeignKey and its adhoc so I'd think there are less circumstances where people have a large number of events tied to a single host, and it was less impact for testing. But since we've altered paths with the `db_constraint` field I'd be ok with also changing this to SET_NULL we think its warranted.